### ISSUE TITLE
cannot read property 'getScrollResponder' of null

### ISSUE THREAD

**Issue by sandeepjrs** [jemise111/react-native-swipe-list-view#181]
![screenshot_2018-01-10-12-34-40-1803030797](https://user-images.githubusercontent.com/14137376/34762904-715c5f8e-f60f-11e7-841a-9a67daf8366c.png)

For temporary fix I am using try and catch in setScrollEnable()
```
	setScrollEnabled(enable) {
		try {
			this._listView.getScrollResponder().setNativeProps({ scrollEnabled: enable });
		}
		catch(e){
			console.log("Error from 'getScrollResponder'", e)
		}		
	} 
```

But I am not sure wether it right way do or not.

**Comment by sandeepjrs** [comment]
this is only happening sometime and  was crashing the app in the production.

**Comment by jemise111** [comment]
Hey @sandeepjrs, in the last issue / PR you submitted you were crashing on the line `this._listView.setNativeProps` so we added the `getScrollResponder()` but now that seems to be crashing too so for some reason your `_listView` ref is disappearing.

Do you have any more code you could share about how you're rendering the SwipeListView?

**Comment by sandeepjrs** [comment]
Sorry For delay.

this is how I am using 
```

<SwipeListView
            useFlatList
            keyExtractor={item => item.id}
           // stopLeftSwipe={true}
            disableRightSwipe={true}
            rightOpenValue={-120}
            swipeToOpenPercent={5}
            swipeToClosePercent={5}
            
            data={this.props.data}
            renderItem={({ item, rowMap}) =>           
                  <TouchableHighlight>
                  
                  </TouchableHighlight> 
            
            renderHiddenItem={(item, rowMap) => (
              
              <TouchableOpacity>

                </TouchableOpacity>              
            )}          
          />
```

**Comment by jemise111** [comment]
@sandeepjrs For the life of me I cannot reproduce this issue. But I'll take your word for it that somehow your users are losing the `ref` to the listView and causing this crash. I've put in a fix in #185 that should solve this. They might not have their scrolling disabled properly but at least the app won't crash. That's the best I can do for now. Please re open if you have any new issues. Thanks!

Released via `v1.0.5`
